task/build-paketo-ubi-builder/OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs: https://go.k8s.io/owners
+approvers:
+  - build-team
+reviewers:
+  - build-team
```

Comment:
@cmoulliard would you changing the owner to yourself?

Reply:
I can do that temporarily but such tasks should be managed by the buildpack team and not me

Reply:
Who is the buildpack team?

Reply:
Fixed with commit: a217e209